test/fixtures/test-runner/plan/timeout-wait-true.mjs

```diff
@@ -0,0 +1,17 @@
+import test from 'node:test';
+
+test('should pass when assertions are eventually met', async (t) => {
+  t.plan(1, { wait: true });
+
+  setTimeout(() => {
+    t.assert.ok(true);
+  }, 250);
+});
```

Comment:
How is this different than the first test of `test/fixtures/test-runner/plan/timeout-options.mjs`?

Reply:
Good catch @atlowChemi, I will remove `test/fixtures/test-runner/plan/timeout-options.mjs`.
I split it into two files, but I forgot to remove the original one 😬